api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -8770,4 +8770,205 @@
 		<!-- visit_attribute_type has not null constraint on uuid -->
 		<!-- visit_type has not null constraint on uuid -->
 	</changeSet>
+	
+	<changeSet id="20151006-1530" author="bahmni">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="order_set" />
+			</not>
+		</preConditions>
+		<comment>
+			Create order_set table
+		</comment>
+		<createTable tableName="order_set">
+			<column name="order_set_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="operator" type="varchar(50)">
+				<constraints nullable="false"/>
+			</column>
+			<column name="name" type="varchar(255)">
+				<constraints nullable="false"/>
+			</column>
+			<column name="description" type="varchar(1000)"/>
+			<column name="creator" type="int">
+				<constraints nullable="false"/>
+			</column>
+			<column name="date_created" type="DATETIME">
+				<constraints nullable="false"/>
+			</column>
+			<column defaultValueBoolean="false" name="retired" type="BOOLEAN">
+				<constraints nullable="false"/>
+			</column>
+			<column name="retired_by" type="int"/>
+			<column name="date_retired" type="DATETIME"/>
+			<column name="retire_reason" type="varchar(255)"/>
+			<column name="changed_by" type="int"/>
+			<column name="date_changed" type="DATETIME"/>
+			<column name="uuid" type="char(38)">
+				<constraints unique="true"/>
+			</column>
+		</createTable>
+		<addForeignKeyConstraint constraintName="order_set_creator"
+								 baseTableName="order_set" baseColumnNames="creator"
+								 referencedTableName="users" referencedColumnNames="user_id" />
+		<addForeignKeyConstraint constraintName="order_set_retired_by"
+								 baseTableName="order_set" baseColumnNames="retired_by"
+								 referencedTableName="users" referencedColumnNames="user_id"/>
+		<addForeignKeyConstraint constraintName="order_set_changed_by"
+								 baseTableName="order_set" baseColumnNames="changed_by"
+								 referencedTableName="users" referencedColumnNames="user_id"/>
+	</changeSet>
+	<changeSet id="20151006-1537" author="bahmni">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="order_set_member" />
+			</not>
+		</preConditions>
+		<comment>
+			Create order_set_member table
+		</comment>
+		<createTable tableName="order_set_member">
+			<column name="order_set_member_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="order_type" type="int">
+				<constraints nullable="false"/>
+			</column>
+			<column name="order_template" type="text"/>
+			<column name="order_set_id" type="int">
+				<constraints nullable="false"/>
+			</column>
+			<column name="sort_weight" type="int">
```

Comment:
Yes, that is what I am saying.

(Basically, sort_weight is the name we use in openmrs by convention for a
double column that is used to sort a set. Having an indexed list handled by
hibernate is a new thing for OpenMRS and on a previous call Burke and I had
agreed we could use the name sequence_number for this column, and that will
be our new convention.)

On Fri, Feb 26, 2016 at 9:29 AM, ys-achinta <notifications@github.com>
wrote:

> In api/src/main/resources/liquibase-update-to-latest.xml
> <https://github.com/openmrs/openmrs-core/pull/1700#discussion_r54273937>:
>
> > +		</preConditions>
> > +		<comment>
> > +			Create order_set_member table
> > +		</comment>
> > +		<createTable tableName="order_set_member">
> > +			<column name="order_set_member_id" type="int" autoIncrement="true">
> > +				<constraints primaryKey="true" nullable="false"/>
> > +			</column>
> > +			<column name="order_type" type="int">
> > +				<constraints nullable="false"/>
> > +			</column>
> > +			<column name="order_template" type="text"/>
> > +			<column name="order_set_id" type="int">
> > +				<constraints nullable="false"/>
> > +			</column>
> > +			<column name="sort_weight" type="int">
>
> So @djazayeri <https://github.com/djazayeri> are u suggesting that i
> should rename this list-index from sort_weight to sequence_number, and use
> integer
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openmrs/openmrs-core/pull/1700/files#r54273937>.
>